pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -177257,8 +177255,7 @@ self: {
        sha256 = "0r8payj72akz951jmv7frllj8cxpwgjyci7gad7250s7nmnd9f1s";
        description = "Interface for JavaScript that works with GHCJS and GHC";
        license = lib.licenses.mit;
-       hydraPlatforms = lib.platforms.none;
-       broken = true;
+       badPlatforms = lib.platforms.linux;
```

Comment:
Looks, like you didn’t rerun the script?

Reply:
Oops